manifests/hourly.pp

```diff
@@ -16,19 +16,27 @@
   Enum['present','absent'] $ensure = 'present'
 ) {
 
+  $manage_cron_hourly = $::logrotate::manage_cron_hourly
+
   $dir_ensure = $ensure ? {
     'absent'  => $ensure,
     'present' => 'directory'
   }
 
+  $cron_ensure = $manage_cron_hourly ? {
+    true  => $ensure,
+    false => 'absent'
+  }
+
   file { "${logrotate::rules_configdir}/hourly":
     ensure => $dir_ensure,
     owner  => 'root',
     group  => 'root',
     mode   => $logrotate::rules_configdir_mode,
   }
-  -> logrotate::cron { 'hourly':
-      ensure => $ensure,
-    }
+  logrotate::cron { 'hourly':
```

Comment:
I do not quite get the logic here. The file is managed, not matter which value  ```$manage_cron_hourly``` has. But the file should only be managed if the variable is true?

Reply:
If manage_cron_hourly is not true then absent must be passed down to logrotate::cron to ensure the cron entry is absent from /etc/cron.hourly.

The whole point of this PR is to allow rotate_every => 'hour' to be used but not have the module manage the associated cron entry for hourly rotations.

Allowing a user to define their own schedule for hourly rotations using puppet::cron and hence dictate when in the hour the rotation occurs, such as on the hour not 17 minutes past the hour which I think is the Ubuntu default.

I would much rather switch this whole module to use puppet::cron and then allow a variable such as logrotate::cron_hourly_minute be used to control when an hourly cron job is run.

But I believe that the adoption of puppet::cron is a larger change than just this PR.